monkey/infection_monkey/post_breach/post_breach_handler.py

```diff
@@ -19,65 +17,34 @@ class PostBreach(object):
     """
     def __init__(self):
         self.os_is_linux = not is_windows_os()
-        self.pba_list = self.config_to_pba_list(infection_monkey.config.WormConfiguration)
+        self.pba_list = self.config_to_pba_list()
 
     def execute(self):
         """
         Executes all post breach actions.
         """
         for pba in self.pba_list:
-            pba.run(self.os_is_linux)
+            pba.run()
         LOG.info("Post breach actions executed")
 
     @staticmethod
-    def config_to_pba_list(config):
+    def config_to_pba_list():
         """
-        Returns a list of PBA objects generated from config.
-        :param config: Monkey configuration
+        Passes config to each post breach action class and aggregates results into a list.
         :return: A list of PBA objects.
         """
         pba_list = []
-        pba_list.extend(PostBreach.get_custom_PBA(config))
-
+        pba_files = get_pba_files()
+        # Go through all of files in ./actions
+        for pba_file in pba_files:
+            # Import module from that file
+            module = importlib.import_module(PATH_TO_ACTIONS + pba_file)
+            # Get all classes in a module
+            pba_classes = [m[1] for m in inspect.getmembers(module, inspect.isclass)
+                           if ((m[1].__module__ == module.__name__) and getattr(m[1], "get_pba", False))]
```

Comment:
instead of getattr, explictly write the check you want, which is checking subclass. 
If they all need to implement get_pba, you have a class that has a method get_pba.
Then you can use `issubclass` and avoid worrying if they support get_pba or anything else